jwst/background/background_step.py

```diff
@@ -58,13 +58,38 @@ def process(self, input, bkg_list):
                 result = background_sub.subtract_wfss_bkg(
                                 input_model, bkg_name, wlrange_name)
                 result.meta.cal_step.back_sub = 'COMPLETE'
-
             else:
+                # check if input data is NRS_IFU
+                tolerance = 1.0e-15
+                result = input_model.copy()
```

Comment:
This copy operation could be moved to the "else" portion of the if-block in which the step is actually called, i.e. down at line 89 or so. That way the copy is only produced if/when we really need it. With it here, the copy gets performed before we even know if we'll need it and hence can get left laying around unused (if we don't need it).